openmetadata-ui/src/main/resources/ui/src/utils/TableUtils.tsx

```diff
@@ -165,7 +165,7 @@ export const getConstraintIcon = (constraint = '', className = '') => {
 
   return (
     <PopOver
-      className={classNames('tw-absolute tw-left-1', className)}
+      className={classNames(className)}
```

Comment:
do we need classNames here?

Reply:
Yes I am sending classname from EntityTable component